tools/add_pre_post_processing_to_model.py

```diff
@@ -130,27 +130,29 @@ def superresolution(model_file: Path, output_file: Path):
     pipeline.add_post_processing(
         [
             Squeeze([0, 1]),  # remove batch and channels dims from Y'
-            FloatToImageBytes(name="Yout_to_bytes"),  # convert Y' to uint8 in range 0..255
+            FloatToImageBytes(name="Y1_uint8"),  # convert Y' to uint8 in range 0..255
+
             # Resize the Cb values (output 1 from PixelsToYCbCr)
-            (
-                Resize((h_out, w_out), "HW"),
-                [IoMapEntry(producer="PixelsToYCbCr", producer_idx=1, consumer_idx=0)],
-            ),
+            (Resize((h_out, w_out), "HW"),
+             [IoMapEntry(producer="PixelsToYCbCr", producer_idx=1, consumer_idx=0)]),
+
             # the Cb and Cr values are already in the range 0..255 so multiplier is 1. we're using the step to round
             # for accuracy (a direct Cast would just truncate) and clip (to ensure range 0..255) the values post-Resize
-            FloatToImageBytes(multiplier=1.0, name="Resized_Cb"),
+            FloatToImageBytes(multiplier=1.0, name="Cb1_uint8"),
+
             (Resize((h_out, w_out), "HW"), [IoMapEntry("PixelsToYCbCr", 2, 0)]),
-            FloatToImageBytes(multiplier=1.0, name="Resized_Cr"),
+            FloatToImageBytes(multiplier=1.0, name="Cr1_uint8"),
+
             # as we're selecting outputs from multiple previous steps we need to map them to the inputs using step names
             (
                 YCbCrToPixels(layout="BGR"),
                 [
-                    IoMapEntry("Yout_to_bytes", 0, 0),  # uint8 Y' with shape {h, w}
-                    IoMapEntry("Resized_Cb", 0, 1),  # uint8 Cb'
-                    IoMapEntry("Resized_Cr", 0, 2),  # uint8 Cr'
+                    IoMapEntry("Y1_uint8", 0, 0),  # uint8 Y' with shape {h, w}
+                    IoMapEntry("Cb1_uint8", 0, 1),
+                    IoMapEntry("Cr1_uint8", 0, 2),
                 ],
             ),
-            ConvertBGRToImage(image_format="jpg"),  # jpg or png are supported
+            ConvertBGRToImage(image_format="png"),  # jpg or png are supported
```

Comment:
do we need to update the test to handle a png instead?

Reply:
Yes.